### PR TITLE
Beecoin store jumpsuits change to jumpskirts based on active character preference.

### PR DESCRIPTION
**Pull request by CydiaLamiales**
Makes the jumpsuits in the beecoin store change into jumpskirts depending on the active character uniform preference.
## Why It's Good For The Game
More customization good.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://user-images.githubusercontent.com/110184118/219882429-c3ee25d0-b326-48b8-a4c5-e88cbab5eac6.mp4

Couldn't really get testing footage of spawning in with a skirt because I don't have a local database set up.

</details>

## Changelog
:cl:
add: Made the jumpsuits in the beecoin store change into jumpskirts depending on the active character preference
add: Added a unique description to jumpskirts
/:cl:

### CODE REVIEW

File: code/modules/client/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -14,54 +13,80 @@
...
 /datum/gear/uniform/color/grey
 	display_name = "grey jumpsuit"
 	path = /obj/item/clothing/under/color/grey
+	skirt_display_name = "grey jumpskirt"
+	skirt_path = /obj/item/clothing/under/color/jumpskirt/grey
 
 /datum/gear/uniform/color/blue
 	display_name = "blue jumpsuit"
 	path = /obj/item/clothing/under/color/blue
+	skirt_display_name = "grey jumpskirt"
```
Comment (imsynthex) [inline code/modules/client/loadout/loadout_uniform.dm]:
```suggestion
	skirt_display_name = "blue jumpskirt"
```

**Review by imsynthex — CHANGES_REQUESTED** [review]

File: code/modules/client/loadout/loadout_uniform.dm (hunk 1 above)
Comment (CydiaLamiales) [reply code/modules/client/loadout/loadout_uniform.dm]: Huh, how did that even happen

**Comment by PowerfulBacon** [comment]
Frozen due to @itsmeow 's preference rework

**Comment by itsmeow** [comment]
No don't do that, it'll take loads of time and it's not much work to implement

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**